persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -14,6 +14,7 @@ CREATE TABLE Conversation (
     muted_time INTEGER DEFAULT 0 NOT NULL,
     last_modified_date TEXT NOT NULL,
     last_notified_message_date TEXT,
+    last_seen_date TEXT,
```

Comment:
We should rename this to `last_read_date` as well it confusing to use different names in logic and in the persistence module.

Reply:
thanks for having a sharp eye ;p